Rubberduck.Resources/RubberduckUI.Designer.cs

```diff
@@ -2265,37 +2265,34 @@ public class RubberduckUI {
                 return ResourceManager.GetString("MakeActiveWorkbookReferenceExplicit", resourceCulture);
             }
         }
-
+        
         /// <summary>
-        ///   Looks up a localized string similar to Make ActiveWorkbook reference explicit.
+        ///   Looks up a localized string similar to Member Attribute Recovery Failure .
```

Comment:
Does this mean that the .resx.cs build is working again, or did you do these by hand?

Reply:
Nope. It did it by itself.